paths/jobs/remove_keys.yaml

```diff
@@ -46,6 +31,26 @@ x-code-samples:
     --project_id <project_id> \
     --id <id> \
     --branch my-feature-branch \
-    --translation_key_ids "abcd1234cdef1234abcd1234cdef1234" \
+    --data '{"branch": "my-feature-branch", "translation_key_ids": ["abcd1234cdef1234abcd1234cdef1234"]}' \
     --access_token <token>
-x-cli-version: '2.5'
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        type: object
+        title: job/keys/delete/parameters
+        required:
+          - name
```

Comment:
what is `name`?

Reply:
thanks for checking. should be `translation_key_ids`